pypuck/pypuck.py

```diff
@@ -101,11 +101,98 @@ def attendance(regular=True, playoffs=True, season=None):
     Examples
     --------
     >>> from pypuck import pypuck
-    >>> pypuck.attendance(regular=True, playoffs=True, season='2019-02')
+    >>> pypuck.attendance(regular=True, playoffs=True, start_season=2000, end_season=2019)
         
     ...
     """
-    pass
+    
+    # Specify the URL
+    url= 'https://records.nhl.com/site/api/attendance'
+
+    # Make the API request
+    page = requests.get(url)
+
+    df = pd.DataFrame(page.json()['data']).sort_values(by=['seasonId'])
+
+    # Check the response code is valid - i.e. the API didn't fail
+    helpers.check_response_code(page.status_code)
+
+    df= df.fillna(0)
+    df.playoffAttendance = df.playoffAttendance.astype(int)
+    df.regularAttendance = df.regularAttendance.astype(int)
+    df = df.rename(columns={'regularAttendance': 'regular',
+                       'playoffAttendance': 'playoff'})
+  
+    # set start seaon and end season to default value if none
+    if pd.isnull(start_season):
+        start_season = 1975    
+
+    if pd.isnull(end_season):
+        end_season = 2019    
+            
+    # check if a proper input is given
+    if type(regular) != bool:
+      raise Exception('Only boolean value can be accepted')
+
+    if type(playoffs) != bool:
+      raise Exception('Only boolean value can be accepted')
```

Comment:
@chuusan if you wanted I made a helper function for this:

```suggestion
    helpers.check_argument_type(regular, 'regular', bool)
    helpers.check_argument_type(playoffs, 'playoffs', bool)
```